src/webhook/handlers/askingArticleSubmissionConsent.ts

```diff
@@ -86,6 +87,11 @@ const askingArticleSubmissionConsent: ChatbotPostbackHandler = async ({
 
   visitor.event({ ec: 'Article', ea: 'Create', el: 'Yes' }).send();
 
+  await sendReplyTokenCollector(
+    userId,
+    t`I will spend some time processing the ${msgsToSubmit.length} new message(s) you have submitted.`
+  );
+
```

Comment:
I think send `sendReplyTokenCollector` twice in askingArticleSubmission is not that good, because:
1. User may(s/he can choose not to click `OK, proceed.`) do lots of actions to send an article to db
2. If the user choose not to click `OK, proceed.` we'll need to push the message every askingArticleSubmission and askingCooccurrence

Maybe we can just let `REPLY_TIMEOUT` mechanism handle this and move "spend some time processing" wording up to previous step?

Reply:
Staging updated. Now the reply token collector only appears when reply token is about to time out. Before that, loading animation is shown instead.